### PR TITLE
Point people to Matrix room instead of IRC

### PR DESCRIPTION
**Pull request by reuben**
X-DeepSpeech: NOBUILD

### CODE REVIEW

**Comment by reuben** [comment]
I think this is everything that refers to IRC from the repo.

File: SUPPORT.rst [hunk 1]
```diff
@@ -11,7 +11,7 @@ There are several ways to contact us or to get help:
...
 
 #. 
-   `\ **IRC** <https://wiki.mozilla.org/IRC>`_ - If your question is not addressed by either the `FAQ <https://github.com/mozilla/DeepSpeech/wiki#frequently-asked-questions>`_ or `Discourse Forums <https://discourse.mozilla.org/c/deep-speech>`_\ , you can contact us on the ``#machinelearning`` channel on `Mozilla IRC <https://wiki.mozilla.org/IRC>`_\ ; people there can try to answer/help
+   `\ **Matrix chat** <https://matrix.to/#/!VkKaHlfgpCUnNSLuWd:mozilla.org?via=mozilla.org>`_ - If your question is not addressed by either the `FAQ <https://github.com/mozilla/DeepSpeech/wiki#frequently-asked-questions>`_ or `Discourse Forums <https://discourse.mozilla.org/c/deep-speech>`_\ , you can contact us on the ``#machinelearning`` channel on `Mozilla Matrix <https://matrix.to/#/!VkKaHlfgpCUnNSLuWd:mozilla.org?via=mozilla.org>`_\ ; people there can try to answer/help
```
Comment (kdavis-mozilla) [inline SUPPORT.rst]: Just checked this link and I think it doesn't work. But this link
```
https://chat.mozilla.org/#/room/!USFCphMyGNrPSFlRph:mozilla.org
```
seems to work.

File: native_client/CODINGSTYLE.md [hunk 2]
```diff
@@ -25,4 +25,4 @@ File naming
 Doubts
 ======
 
-If in doubt, please ask on our IRC channel, `#machinelearning` on `irc.mozilla.org`.
+If in doubt, please ask on our Matrix chat channel: https://matrix.to/#/!VkKaHlfgpCUnNSLuWd:mozilla.org?via=mozilla.org
```
Comment (kdavis-mozilla) [inline native_client/CODINGSTYLE.md]: Just checked this link and I think it doesn't work. But this link
```
https://chat.mozilla.org/#/room/!USFCphMyGNrPSFlRph:mozilla.org
```
seems to work.

File: SUPPORT.rst (hunk 1 above)
Comment (reuben) [reply SUPPORT.rst]: It works for me, it's just a bit convoluted. But I'll revert to chat.mozilla.org.

**Pull request merged**